apps/system/test/unit/multi_screen_controller_test.js

```diff
@@ -39,12 +52,14 @@ suite('system/MultiScreenController', function() {
   }
 
   setup(function() {
+    realApplications = window.applications;
+    window.applications = MockApplications;
```

Comment:
Hi Luke, after I removed this lines, the ReferenceError happened.

```
Uncaught Error: ReferenceError: applications is not defined (app://system.gaiamobile.org/js/browser_config_helper.js?time=1455704743577:45)    
```

Please see test result in the following link:
https://tools.taskcluster.net/task-inspector/#H9eyedMURu2cfp9HxyYeSQ/2